features/roomlist/impl/src/main/kotlin/io/element/android/features/roomlist/impl/RoomListPresenter.kt

```diff
@@ -176,6 +182,7 @@ class RoomListPresenter @Inject constructor(
             derivedStateOf {
                 when {
                     currentSecurityBannerDismissed -> SecurityBannerState.None
+                    needsSlidingSyncMigration -> SecurityBannerState.NeedsNativeSlidingSyncMigration
```

Comment:
Are you sure that the `NeedsNativeSlidingSyncMigration` banner has higher priority than the `SetUpRecovery` and `RecoveryKeyConfirmation` banner?

It will fully hide the two latter, since if `NeedsNativeSlidingSyncMigration` is dismissed, `currentSecurityBannerDismissed` will be true and the other banner will never be displayed again (until the user logout and login again).

Reply:
Yes, you're right, this one should have the lowest priority.